src/com/lftechnology/java/training/alina/basic/dayname/DayInWord.java

```diff
@@ -5,80 +5,79 @@
 import java.util.logging.Logger;
 
 /**
- * Write a program called PrintDayInWord, which prints “Sunday”, “Monday”, ...
- * “Saturday” if the int variable "day" is 0, 1, ..., 6, respectively.
- * Otherwise, it shall print “Not a valid day”.
+ * Write a program called PrintDayInWord, which prints “Sunday”, “Monday”, ... “Saturday” if the int variable "day" is 0, 1, ..., 6,
+ * respectively. Otherwise, it shall print “Not a valid day”.
  * 
  * @author Alina Shakya <alinashakya@lftechnology.com>
  *
  */
 public class DayInWord {
 
-	private static final Logger LOGGER = Logger.getLogger(DayInWord.class.getName());
+    private static final Logger LOGGER = Logger.getLogger(DayInWord.class.getName());
 
-	public static void main(String[] args) {
-		try(Scanner scanner = new Scanner(System.in)) {
-			displayDayName(scanner);
-		} catch (Exception e) {
-			LOGGER.log(Level.WARNING, "Exception Message : {0}", new Object[] {e.getMessage()});
-			throw new RuntimeException(e);
-		}
-	}
+    public static void main(String[] args) {
+        try (Scanner scanner = new Scanner(System.in)) {
+            displayDayName(scanner);
+        } catch (Exception e) {
+            LOGGER.log(Level.WARNING, "Exception Message : {0}", new Object[] { e.getMessage() });
+            throw new RuntimeException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)